exporter/osg/osgbake.py

```diff
@@ -47,6 +48,43 @@ def collectValues():
             for block in morph_info:
                 morph_info[block].append(block.value)
 
+    def evaluateActiveShapeKeys(time, times):
+        # Get effective shapes according to current time
+        # and evaluate the factor for each
+        value = 0.0
+        if time > times[-1]: # after the last key_block time
+            previous_block = shape.key_blocks[-1]
+            next_block = None
+        elif time < times[0]: # before the first key_block time
+            previous_block = None
+            next_block = shape.key_blocks[0]
+            value = 1.0
+        else:
+            # general case with index in [0:16] and blending two key_blocks
+            index = bisect.bisect(times, time) - 1
+            previous_block = shape.key_blocks[index]
+            next_block = shape.key_blocks[index + 1]
+            #TODO implement others interpolations (Cardinal, Bspline, Catmull-Rom) or find a way to
+            # get the value with Blender API
+            value = (time - previous_block.frame) / (next_block.frame - previous_block.frame)
```

Comment:
would be nice to be a bit more consistant on variable namings; `frame` vs `time` vs `keyTime`